src/shader.vert

```diff
@@ -37,8 +38,12 @@ void main() {
     Point p = points[gl_InstanceIndex];
     vec4 out_Pos = view_proj * model_transform * vec4(p.position_x, p.position_y, p.position_z, 1.0);
 
+    #ifdef COLORED
     out_Color = vec3(p.color_r, p.color_g, p.color_b);
-
+    #else
+    out_Color = vec3(p.position_x, p.position_y, p.position_z);
+    #endif
```

Comment:
Yes that is the case long term, but for opd replay specifically I was told to "not worry about colors" for now. Without this it will be hard to tell that the particles are moving.

Reply:
OK, but it seems like you've gone out of your way to have a special case that doesn't support colors when you already had color support to begin with.

> Without this it will be hard to tell that the particles are moving.

I don't understand. I would expect it to be easier to see movement if each point has a constant color rather than changing as it's position changes.